resources/js/components/StackedChart.vue

```diff
@@ -9,10 +9,22 @@
         <default-button size="xs" class="mr-2" @click="reloadPage()" v-show="buttonReload">
           <icon-refresh />
         </default-button>
-        <default-button size="xs" class="mr-2" component="a" :href="externalLink" :target="externalLinkIn" v-show="btnExtLink">
+        <default-button
+          size="xs"
+          class="mr-2"
+          component="a"
+          :href="externalLink"
+          :target="externalLinkIn"
+          v-show="btnExtLink"
+        >
           <icon-external-link />
         </default-button>
-        <select-control size="xxs" @change="handleFilterChanged" :selected="advanceFilterSelected" v-show="showAdvanceFilter">
+        <select-control
+          size="xxs"
+          @change="handleFilterChanged"
+          :selected="advanceFilterSelected"
```

Comment:
We moved to `v-model` and `defineModel()` on Nova 5, so I'm unsure if `:selected` does anything here.